hasjob/views/job_alerts.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+
+
+from flask import abort, redirect, render_template, request, url_for, g, flash
+from flask_mail import Message
+from flask_rq import job
+from premailer import transform as email_transform
+from pyisemail import is_email
+from html2text import html2text
+from baseframe import _
+from .. import app, mail
+from ..models import (db, JobPostSubscription, Filterset)
+
+
+@job('hasjob')
+def send_confirmation_email_for_job_alerts(to_address, token):
+    msg = Message(subject=u"Please confirm your email to receive alerts on new jobs", recipients=[to_address])
+    html = email_transform(render_template('job_alert_email_confirmation.html.jinja2', token=token))
+    msg.html = html
+    msg.body = html2text(html)
+    mail.send(msg)
+
+
+@app.route('/subscribe_to_job_alerts', subdomain='<subdomain>', methods=['POST'])
+@app.route('/subscribe_to_job_alerts', methods=['POST'])
```

Comment:
I don't think we should pollute the top-level namespace with these URLs. Park them in a path. `/api/1/` should work. See #145 for rationale.